### PR TITLE
Update docs for correctness

### PR DESCRIPTION
**Pull request by dhinklexo**
## Types of changes

- [ ] Documentation only (no changes to either `lib/` or `test/` files)
- [ ] Bug fix (non-breaking change which fixes an issue. you didn't modify existing tests)
- [ ] New feature (non-breaking change which adds functionality. you added at least one new test)
- [ ] Breaking change (fix or feature that would cause existing functionality to change. you had to modify existing tests.)

## Checklist:

- [ ] I have read the [**CONTRIBUTING**](https://github.com/xogroup/toki/blob/master/.github/CONTRIBUTING.md) document.
- [ ] My code follows the [Hapi.js style guide](https://github.com/hapijs/contrib/blob/master/Style.md).
- [ ] I have updated the documentation as needed.
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.

### CODE REVIEW

File: API.md [hunk 1]
```diff
@@ -24,79 +23,36 @@
...
 //intantiate toki
 const Toki = require('toki');
-const toki = new Toki({
-    router : routerInstance
-});
 
-//wait for toki to be ready
 toki.on('ready', ()=>{
-    //toki is open for buisness (aka do your thing)    
+    //Ready
```
Comment (WesTyler) [inline API.md]: Toki is where fun goes to die. All work and no play makes Toki efficient and brutal. :P

File: Examples.md [hunk 2]
```diff
@@ -1,218 +1,101 @@
 # Examples
 
-Here's a BIG BEAUTIFUL example showing all __toki__ pieces working together.
-
+These examples primarily focus on Toki bridges, methods and configs. To see examples of Toki in action, please check out the [Toki Reference Implementation](https://github.com/xogroup/toki-reference)
```
Comment (WesTyler) [inline Examples.md]: Noice

File: README.md [hunk 3]
```diff
@@ -1,5 +1,6 @@
-# toki
-A configuration based orchestration rules engine with support for custom plugins.  
+# Toki
+
+> A configuration based orchestration rules engine that's protocol agnostic.
```
Comment (WesTyler) [inline README.md]: Maybe "A protocol-agnostic, configuration-based orchestration rules engine." ?

File: README.md [hunk 4]
```diff
@@ -11,102 +12,37 @@ Lead Maintainer: [Cesar Hernandez](https://github.com/cesarhq)
...
-A logging interface that can be instantiated beforehand with any standard logging library. 
-__toki__ will require the toki-logger instance to log events and actions related to it's own lifecycle and request/rule execution.
++ *Configuration based* using version-able JSON files. Easy to write, easy to deploy.
 
-```Javascript
-//setup logger
-const winston = require('winston');
-const TokiLogger = require('toki-logger');
-const logger = TokiLogger(winston);
++ *Bring your own HTTP framework* so you can use Express, Koa, Hapi or whatever else floats your boat.
```
Comment (WesTyler) [inline README.md]: Okay I guess we can allow a *little* bit of fun flavor into Toki haha

File: README.md [hunk 5]
```diff
@@ -11,102 +12,37 @@ Lead Maintainer: [Cesar Hernandez](https://github.com/cesarhq)
...
+To allow Toki to be used by any http framework, Toki uses small modules we call 'bridges'. The bridge appropriate for your webserver framework is where you should start:
 
-- put together your configuration as per [toki configuration](./RULESENGINE.md#configuration)
++ [Hapi](https://github.com/xogroup/toki-hapi-bridge)
 
-- build your action handler as per [toki action handler](./RULESENGINE.md#how-to-implement-my-very-own-action-handler) listed steps
+_Don't see a bridge you need? Please feel free to contribute one. Bridges are easy to write, we promise!_
 
-- setup __toki__ dependencies
+You can also peek at our [reference implementation](https://github.com/xogroup/toki-reference), which provides a fully setup and ready to roll Toki implementation out of the box that's heavily commented and explained.
```
Comment (WesTyler) [inline README.md]: I like this a lot.

File: API.md (hunk 1 above)
Comment (dhinklexo) [inline API.md]: NO FUN FOR YOU

**Pull request merged**